src/it/resources/apis.yaml

```diff
@@ -45,7 +45,7 @@ paths:
       tags:
         - pets
       responses:
-        '201':
+        '200':
```

Comment:
This is the apis.yaml in the integration tests path `it` and not in the source coee

Reply:
Yes I understood it, but not knowing if it was part of the PR I changed it anyway, if it's not good I report everything as it was

Reply:
The point is that this PR is a noop at the moment, you should modify the source too